app/controllers/ems_common.rb

```diff
@@ -337,6 +337,7 @@ def button
       when "security_group_tag"               then tag(SecurityGroup)
       end
 
+      return if params[:pressed].include?("tag") && !params[:pressed].start_with?("host", "vm", "miq_template")
```

Comment:
are the values for "pressed" you are detecting here `%w(host_tag vm_tag miq_template_tag)` ?

Can you, please, make it an explicit list instead of matching the beginnings/ends?

It will be much easier to refactor if we know the values right here and this will have to be refactored.

Thx!